ex3.html

```diff
@@ -0,0 +1,558 @@
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>						
+										</select>
+									</li>
+									<li>Tooling
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>						
+										</select>
+									</li>
+									<li>Libraries
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>		
+										</select>
+									</li>
+									<li>Promises
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>				
+										</select>
+									</li>
+									<li>Browser Engines
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>			
+										</select>
+									</li>
+									<li>Frameworks
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>			
+										</select>
+									</li>
+								</ul>
+
+							</fieldset>
+
+
+
+							<fieldset>
+								<legend>NodeJS</legend>
+								<ul>
+									<li>Backend frameworks
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>	
+										</select>
+									</li>
+									<li>Templating
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>
+										</select>
+									</li>
+									<li>DOM Manipulation
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>
+										</select>
+									</li>
+									<li>Unit Testing
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>				
+										</select>
+									</li>
+								</ul>
+
+							</fieldset>
+
+							<h3><a href="ex2.html" target="_top" style="float: right;">SUBMIT</a></h3>
+						</form>
+
+
+						<style type="text/css">
```

Comment:
I know I've brought this up on another PR recently, but I can't remember whose it was. Please note that CSS is render blocking by default. This means that in order to be able to display your page, the browser first needs to a) load the page, b) load any applicable CSS, c) process these two and glue them together before being able to render your page at all.

What this translates into is that the quicker the browser has this info (i.e., the higher you put your styling assets in the DOM), the quicker it can do its stuff and render the page for your users to delight on. More info on this topic [here](https://developers.google.com/web/fundamentals/performance/critical-rendering-path/render-blocking-css).